content/en/entities/V1_NotificationPolicy.md

```diff
@@ -0,0 +1,86 @@
+---
+title: V1::NotificationPolicy
+description: Represents the notification filtering policy of the user.
+menu:
+  docs:
+    parent: entities
+aliases: [
+  "/entities/v1_NotificationPolicy",
+]
+---
+
```

Comment:
I'd perhaps add something somewhere here to indicate that this entity is deprecated?